guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -14,7 +14,8 @@ If you want unlimited hosts to register with the activation key, ensure the *Unl
 . From the *Environment* list, select the environment to use.
 . From the *Content View* list, select a content view to use.
 ifndef::orcharhino[]
-. In the *Repository Sets* tab, ensure only your named repository is enabled.
+. In the *Repository Sets* tab, override repositories to Enabled or Disabled as desired.
+For more information, see {ContentManagementDocURL}Enabling_Red_Hat_Repositories_content-management[Enabling Red Hat repositories] in _{ContentManagementDocTitle}_.
```

Comment:
Wrong kind of repository enabling (I know..)

This should instead link to https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#enabling-and-disabling-repositories-on-activation-key_content-management